core/src/test/java/io/questdb/test/cairo/mv/CreateMatViewTest.java

```diff
@@ -149,6 +149,58 @@ public void testCreateMatViewBaseTableDoesNotExist() throws Exception {
         assertNull(getMatViewDefinition("testView"));
     }
 
+    @Test
+    public void testCreateMatViewCopySymbolCapacity() throws Exception {
+        assertMemoryLeak(() -> {
+            createTable(TABLE1);
+            final String sql = "select ts, k, min(v) as v from " + TABLE1 + " sample by 1h";
+            execute("create materialized view test with base " + TABLE1 + " as (" + sql + ") partition by day");
+            assertSql("column\tsymbolCapacity\nk\t2048\n", "select \"column\", symbolCapacity from (show columns from test) where type = 'SYMBOL'");
+        });
+    }
+
+    @Test
+    public void testCreateMatViewCopySymbolCapacityAlias() throws Exception {
+        assertMemoryLeak(() -> {
+            createTable(TABLE1);
+            final String sql = "select ts, k as kkk, min(v) as v from " + TABLE1 + " sample by 1h";
+            execute("create materialized view test with base " + TABLE1 + " as (" + sql + ") partition by day");
+            assertSql("column\tsymbolCapacity\nkkk\t2048\n", "select \"column\", symbolCapacity from (show columns from test) where type = 'SYMBOL'");
+        });
+    }
+
+    @Test
+    public void testCreateMatViewCopySymbolCapacityExpr() throws Exception {
+        assertMemoryLeak(() -> {
+            createTable(TABLE1);
+            final String sql = "select ts, cast(k || '_' as symbol) as kkk, min(v) as v from " + TABLE1 + " sample by 1h";
```

Comment:
nit: using the same alias as the OG symbol column name might be a good idea
```suggestion
            final String sql = "select ts, cast(k || '_' as symbol) as k, min(v) as v from " + TABLE1 + " sample by 1h";
```

Reply:
fixed in: https://github.com/questdb/questdb/pull/5619/commits/f3428239ab9ad96c7fbb053ed55501d1343966d6